signing/sawtooth_signing/secp256k1_signer.py

```diff
@@ -88,38 +76,11 @@ def _decode_privkey_to_bytes(encoded_privkey, encoding_format='wif'):
     else:
         raise TypeError("unsupported private key format")
 
-    return priv
-
-
-def decode_privkey(encoded_privkey, encoding_format='wif'):
-    """Decodes a provided encoded privkey to a secp256k1 bytes representation
-
-    Args:
-        encoded_privkey (str): An encoded private key string
-        encoding_format (str): The encoded format of the provided
-            private key. Must be either 'wif' or 'hex'.
-
-    Returns:
-        bytes: A private key in native bytes format
-    """
-    return _decode_privkey_to_bytes(encoded_privkey, encoding_format)
-
-
-def _decode_privkey(encoded_privkey, encoding_format='wif'):
-    """
-    Args:
-        encoded_privkey: an encoded private key string
-        encoding_format: string indicating format such as 'wif'
-
-    Returns:
-        private key object useable with this module
-    """
-    priv = _decode_privkey_to_bytes(encoded_privkey, encoding_format)
     return secp256k1.PrivateKey(priv, ctx=__CTX__)
 
 
 def generate_pubkey(privkey):
```

Comment:
One of the intents of this PR is to save the developer from figuring out formats. Consequently the public methods do not expose any format options. Adding a format override to this method would breakdown that pattern. Incidentally, in the near future we may want to drop wif, which would, in turn eliminate the dependency on pybitcointools. This API insulates the callers from that kind of change.